tests/integration/telemetry/main_test.go

```diff
@@ -36,7 +36,9 @@ func TestMain(m *testing.M) {
 		Label(label.CustomSetup).
 		Setup(istio.Setup(&i, nil)).
 		Setup(func(ctx resource.Context) (err error) {
-			ingr = i.IngressFor(ctx.Clusters().Default())
+			for _, cl := range ctx.Clusters() {
+				ingr = append(ingr, i.IngressFor(cl))
```

Comment:
Could we just give `istio.Instance` a `Ingresses()` method?

Reply:
Yeah that might be good too.. I will refactor some tests in a different pr to try to reduce time.. will handle there?